### ISSUE TITLE
Enable writing directly in the text input.

### ISSUE THREAD

**Issue by yisselda** [yisselda/SlackableThemes#59]
We want to be able to edit and paste in a full theme.

**Comment by twummers** [comment]
Hey @yisselda - is the ask here to allow typing directly into the complete (read-only) color input field?  I'm already able to edit individual theme attribute inputs.

**Comment by yisselda** [comment]
@twummers yes it's exactly what you described, to be able to change the theme directly from the complete theme input.

![image](https://user-images.githubusercontent.com/29551766/109559362-8aff1780-7aa8-11eb-9353-3131e18c0f98.png)

The impact of changing from the complete theme input is that it needs to update the corresponding individual theme attribute input as well.

**Comment by twummers** [comment]
Thanks for confirming! Will tackle this issue.

**Comment by yisselda** [comment]
@twummers that works, thank you for taking this on :)
